docs/Reference/CLI/Subcommands/Admin.md

```diff
@@ -520,6 +520,10 @@ Possible values are:
 | `mainnet` | Consensus layer | Production  | Main network.                                              |
 | `minimal` | Consensus layer | Test        | Used for local testing and development networks.           |
 | `prater`  | Consensus layer | Test        | Multi-client testnet.                                      |
+| `kiln`    | Consensus layer | Test        | Multi-client testnet                                       |
```

Comment:
Either include or exclude the ending period for all network descriptions for consistency.